src/services/ImageRepository.cpp

```diff
@@ -300,7 +302,7 @@ static HRESULT CreateDIBFromBitmapSource(_In_ IWICBitmapSource *pToRenderBitmapS
         hr = UIntMult(cbStride, nHeight, &cbImage);
 
     // Extract the image into the HBITMAP    
-    if (SUCCEEDED(hr))
+    if (SUCCEEDED(hr) && pvImageBits)
```

Comment:
Something about `pvImageBits `not being allowed to be `nullptr `in a API call below.